PROJECT_PROGRESSION.md

```diff
@@ -100,7 +101,7 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
 - [ ] Publish Project Charter on website or GitHub
 - [ ] Update legal copyright notice on project website and GitHub
 - [ ] Add OpenJS Foundation logo to project website
-- [ ] Add Project logo to OpenJS Foundation website; update PROJECTS.md file
+- [ ] Add Project logo to OpenJS Foundation website
```

Comment:
This should be "update the README" instead.

Reply:
Added a separate action item to the checklist clarifying this.